system/firewalld.py

```diff
@@ -162,6 +168,29 @@ def remove_source(zone, source):
     fw_zone.update(fw_settings)
 
 ####################
+# interface handling
+#
+def get_interface(zone, interface):
+    fw_zone = fw.config().getZoneByName(zone)
+    fw_settings = fw_zone.getSettings()
+    if interface in fw_settings.getInterfaces():
+       return True
```

Comment:
I agree, that would be less code, but the "if .. in ...: return True else return False" pattern is used in the other methods, too. Personally I would sacrifice "less code" for "keep existing patterns" for better readability and I didn't want to introduce too many changes by also changing the other methods. @maxamillion, what are your preferences?